artview/components/plot_radar.py

```diff
@@ -965,7 +966,7 @@ def setupUi(self):
         self.layout.addWidget(QtGui.QLabel("Vlims"), 3, 0)
         self.layout.addWidget(self.limsButton, 3, 1, 1, 3)
 
-        self.name = QtGui.QLineEdit("Display")
+        self.name = QtGui.QLineEdit("RadarDisplay")
```

Comment:
I like the changes here and grid_radar more explicitly defining the Display used.